src/test/java/study/racinggame/domain/GameResultTest.java

```diff
@@ -0,0 +1,35 @@
+package study.racinggame.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GameResultTest {
+  @Test
+  @DisplayName("pobi가 가장 전진 많이한 경우 승자 확인 테스트")
+  public void winners_승자_pobi() {
+    Car pobiCar = new Car("pobi");
+    Car onenyCar = new Car("oneny");
+    Car twonyCar = new Car("twony");
+
+    pobiCar.forward(() -> true);
+    pobiCar.forward(() -> true);
+
+    List<Car> carList = new ArrayList<>();
+
+    carList.add(pobiCar);
+    carList.add(onenyCar);
+    carList.add(twonyCar);
+
+    Cars cars = new Cars(carList);
+
+    GameResult gameResult = new GameResult();
+    gameResult.addRecord(cars);
+
+    assertThat(gameResult.winners()).isEqualTo("pobi");
```

Comment:
```suggestion
    //given
    pobiCar = POBI_CAR;
    onenyCar = ONENY_CAR;
    twonyCar = TWONY_CAR;
    String expected = pobiCar.name();
    
    // when
    pobiCar.forward(() -> true);
    pobiCar.forward(() -> true);
    
    // then
    GameResult gameResult = new GameResult(new Cars(pobiCar, onenyCar, twonyCar));
    assertThat(gameResult.winners()).isEqualTo(expected);
```
- 메소드로 분리하는 등 추상화하여 테스트하고자 하는 부분이 부각되도록 리팩토링하는건 어떨까요